### ISSUE TITLE
Printing an object that is placed above Zero Z (drop down model is unchecked) always start printing from Zero Z

### ISSUE THREAD

**Issue by StmLord** [Ultimaker/Cura#17554]
### Cura Version

5.5

### Operating System

Mac OS Monterey

### Printer

Custom FFF

### Reproduction steps

1) place an object on bed
2) uncheck drop down model and lift it from bed
3) slice 

### Actual results

both preview and real printing starts at Zero Z axis.

### Expected results

slicing should respect Z height of an Object. 

### Add your .zip and screenshots here ⬇️

![Screenshot 2023-12-08 at 1 11 13 AM](https://github.com/Ultimaker/Cura/assets/66776884/bbfde32a-19bf-4635-811a-e4ad7c556490)
![Screenshot 2023-12-08 at 1 11 41 AM](https://github.com/Ultimaker/Cura/assets/66776884/5051fb8d-03ed-448b-bfa0-4433d33008bf)

**Comment by GregValiant** [comment]
Thanks for the report.
There is a setting bug involved here.

- Disable "Generate Support"
- Go to the Mesh Fixes section and disable "Remove empty first layers".
- Turn supports back on (if you need them).

The bug is that if the setting is going to impact the slice, that it should be visible.
I'll mark this as a duplicate for the Cura team.
![image](https://github.com/Ultimaker/Cura/assets/64202104/f7445af0-ec61-44fd-9995-4de52707dae5)

**Comment by StmLord** [comment]
Yes, thats it, I had to enable the view of "remove empty first layers" setting to disable it and that did the trick.
Thanks
